util/archive_handler_test.go

```diff
@@ -37,5 +37,20 @@ var _ = Describe("ArchiveHandler", func() {
 				os.Remove(mtaArchiveFilePath)
 			})
 		})
+
+		Context("with oversize descriptor size", func() {
+			BeforeEach(func(){
+				setDefaultMaxDescriptorSize(64)
+			})
+			It("must return error for file exceeds max size limit", func(){
+				var mtaArchiveFilePath, _ = filepath.Abs("../test_resources/commands/mtaArchive.mtar")
```

Comment:
Is it for sure that this "../test_resources/commands/mtaArchive.mtar" will exist alway? Maybe some logic would be nice here?